app/server/routes/v1/post.js

```diff
@@ -269,7 +289,7 @@ router.get("/community/:title", async (req, res) => {
  *        '401':
  *          description: Not Authorized. Only creator of post can edit post.
  *        '403':
- *          description: Forbidden. Community can't have more than 3 pinned posts. OR Forbidden. Can't edit creator of a post.
+ *          description: Forbidden. Community can't have more than 3 pinned posts. OR Forbidden. Can't edit tags. Use /api/post/{id}/tags instead. OR Forbidden. Can't edit creator of a post.
```

Comment:
This description seems a little weird to read imo, lets refactor?

Reply:
Swagger doesn't let me use multiple descriptions or responses so I need to include them all in the same description. Do you have any suggestions for how I should write the description?

Reply:
This is a feature that many people on the internet seem to want but Swagger will not implement it.

Reply:
Kk